tests/integration_tests/performance/test_vsock_throughput.py

```diff
@@ -8,85 +8,77 @@
 import logging
 import time
 import concurrent.futures
+
 import pytest
 from conftest import _test_images_s3_bucket
 from framework.artifacts import ArtifactCollection, ArtifactSet
 from framework.matrix import TestMatrix, TestContext
 from framework.builder import MicrovmBuilder
-from framework.statistics import core, consumer, producer, criteria, types
-from framework.utils import CpuMap, CmdBuilder, run_cmd, eager_map, \
-    get_cpu_percent
+from framework.statistics import core, consumer, producer
+from framework.statistics.baseline import Provider as BaselineProvider
+from framework.statistics.metadata import DictProvider as DictMetadataProvider
+from framework.utils import CpuMap, CmdBuilder, run_cmd, get_cpu_percent, \
+    DictQuery
 from framework.utils_cpuid import get_cpu_model_name
 import host_tools.network as net_tools
-import integration_tests.performance.configs.vsock_throughput_test_config as\
-    test_cfg
-
-
-SERVER_STARTUP_TIME = test_cfg.CONFIG["server_startup_time"]
-VSOCK_UDS_PATH = "v.sock"
-
 
-def measurements():
-    """Define the produced measurements for VSOCK workloads."""
-    return [types.MeasurementDef(test_cfg.THROUGHPUT,
-                                 test_cfg.THROUGHPUT_UNIT),
-            types.MeasurementDef(test_cfg.DURATION, test_cfg.DURATION_UNIT),
-            types.MeasurementDef.cpu_utilization_vmm(),
-            types.MeasurementDef.cpu_utilization_vcpus_total()]
 
+# Measurements units.
+CONFIG_RAW_FILE = os.path.join(
+    os.path.dirname(__file__),
```

Comment:
What about defining a base folder that contains the config files and just do a one liner?

```
# snip
CFG_LOC = Path("/well/known/location")
# end of snip

CONFIG = json.load(open(CFG_LOC / "vsock_throughput_test_config.json"))
```

Reply:
Done.